packages/core/src/navigation/navigation-item.element.ts

```diff
@@ -109,7 +112,6 @@ export class CdsNavigationItem extends LitElement implements FocusableItem {
 
   protected updated(props: PropertyValues) {
     super.updated(props);
-    this.disabled ? this.setAttribute('tabindex', '-1') : this.removeAttribute('tabindex');
```

Comment:
Are we keeping `tabindex` on disabled items?

Reply:
Yes, it prevents them from being tabbable.